core/tauri-utils/src/config.rs

```diff
@@ -1002,6 +1002,9 @@ pub struct WindowConfig {
   /// The window webview URL.
   #[serde(default)]
   pub url: WindowUrl,
+  /// The webview proxy using by wry
```

Comment:
Maybe something like this? (you need to run `carg build` to regenerate the json files)
```suggestion
  /// The proxy URL used by the webview for all network requests.
```